docs/working-remotely-ssh.md

```diff
@@ -0,0 +1,49 @@
+# Working remotely with Visual Studio Code
+
+It is possible to work remotely on a machine containing the required opensafely software using Visual Studio Code.
+This may be useful in cases where it is not possible to install all of the required sofware to run opensafely on your local machine, or where running this required software severly affects the performance of your local machine. 
```

Comment:
nit: "severly" should be "severely"

Reply:
Maybe worth clarifying that there is a requirement that the server requires that its network connection allows connections on a port open for SSH, and that the client machine is able to make connections to that port? Some users might be on networks with a firewall that they do not control.

Also maybe worth emphasising that you should have permission from whoever is running the server to do so, if that is not you? In principle, you would require admin access to configure this, but still.